ibtk/include/ibtk/FEDataManager.h

```diff
@@ -1012,14 +1041,15 @@ class FEDataManager : public SAMRAI::tbox::Serializable
 
     /*!
      * Collect all of the active elements which are located within a local
-     * Cartesian grid patch grown by the specified ghost cell width.
+     * Cartesian grid patch grown by a ghost width of 1 (like
+     * IBTK::LEInteractor::getMinimumGhostWidth(), we assume that IB points
+     * are allowed to move no more than one cell width between regridding
```

Comment:
Can we _easily_ encode this assumption about the # of cells points can move between regridding operations somewhere, perhaps as a constant member variable? (If it is not easy to do, let's not worry about it for now.)

Reply:
Not really; see below.